src/modules/repository/implementation/sequelize/sequelize-repository.js

```diff
@@ -272,14 +272,22 @@ class SequelizeRepository {
                 blockchain_id: {
                     [Sequelize.Op.eq]: blockchain,
                 },
+                last_seen: {
+                    [Sequelize.Op.gte]: Sequelize.col('last_dialed'),
```

Comment:
This is the filtering of the non active nodes.

When dialing peers, we only update last seen if dial was successful. If not, we only update last seen.

Reply:
> This is the filtering of the non active nodes.
> 
> When dialing peers, we only update last seen if dial was successful. If not, we only update last seen.

Makes sense, but then we will filter out all nodes that didn't responded once. Do we want to do it like this?

Reply:
We will filter out peers that didn't respond the last time we contacted them. Since we dial every hour, the list should be fairly up to date. We don't expect nodes to go down so often